src/main/g8/src/main/resources/application.conf

```diff
@@ -0,0 +1,21 @@
+twitter {
+  consumer {
+    key = "my-consumer-key"
+    secret = "my-consumer-secret"
+  }
+  access {
+    key = "my-access-key"
+    secret = "my-access-secret"
+  }
+
+  rest {
```

Comment:
setting not needed to run twitter4s. It should be used only if you need to change the URLs because of some proxy